packages/devtools/lib/src/ui/dragscroll.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:html';
+
+import '../ui/elements.dart';
+
+void listenForDragScrolling(CoreElement element) {
+  final dragged = element.element;
+
+  num lastX;
+  num lastY;
+  bool clicked = false;
+
+  dragged.addEventListener(
```

Comment:
nit:
use
.onMouseDown.listen((event) {

it is more darty.

Reply:
done.